90metalmdsquash/metal-md-disks.sh

```diff
@@ -49,12 +49,15 @@ pave
 # Die if no viable disks are found; otherwise continue to disk creation functions.
 if [ ! -f /tmp/metalsqfsdisk.done ]; then
     md_disks=()
-    for disk in $(seq 1 $metal_disks); do
-        md_disk=$(metal_resolve_disk $(metal_scand $disk) $metal_disk_small)
-        md_disks+=( $md_disk )
+    disks="$(metal_scand)"
+    IFS=" " read -r -a pool <<< "$disks"
+    for disk in "${pool[@]:0:$metal_disks}"; do
```

Comment:
On second thought, this should look through the entire pool.

In the event that `metal_resolve_disk` doesn't like one of the disks, we have to keep scanning. With how it's currently setup, it'll only try a few disks.